packages/ui/src/theme/tokens/types/designToken.ts

```diff
@@ -41,7 +24,9 @@ export type WebDesignToken<ValueType = any> = {
    * `var(--amplify-colors-font-primary)`
    */
   toString(): string;
-} & DesignToken<ValueType>;
+
+  value: ValueType;
```

Comment:
Include `value: ValueType` directly on `WebDesignToken` and remove the intersection type to prevent extension of `ValueType`